mentors/python/k.json

```
@@ -6,5 +6,13 @@
     "link_url" : null,
     "avatar_url" : null,
     "bio" : "import this"
+  },
+  {
+    "github_username": "khoivan88",
+    "name": "Khoi Van",
+    "link_text": "My github",
```

```suggestion
    "link_text": "My GitHub",
```

Just a small capitalization fix.

---

Thank you for your suggestion!